### PR TITLE
Support for CCTray XML format

### PR DESCRIPTION
**Pull request by fkorotkov**
Fixes https://github.com/cirruslabs/cirrus-ci-docs/issues/588

### CODE REVIEW

**Comment by fkorotkov** [comment]
Checked with CCMenu

image

@bekh6ex  PTAL

File: docs/guide/writing-tasks.md [hunk 1]
```diff
@@ -848,3 +848,14 @@ Here is how Cirrus CI's badge can be embeded in a Markdown file:
 ```markdown
 [![Build Status](https://api.cirrus-ci.com/github/<USER OR ORGANIZATION>/<REPOSITORY>.svg)](https://cirrus-ci.com/github/<USER OR ORGANIZATION>/<REPOSITORY>)
 ```
+
+## CCTray XML
+
+Cirrus CI support exporting information about the latest repository builds via [CCTray XML format](https://cctray.org/).
```
Comment (RDIL) [inline docs/guide/writing-tasks.md]:
```suggestion
Cirrus CI supports exporting information about the latest repository builds via the [CCTray XML format](https://cctray.org/).
```

File: docs/guide/writing-tasks.md [hunk 2]
```diff
@@ -848,3 +848,14 @@ Here is how Cirrus CI's badge can be embeded in a Markdown file:
...
+Use the following URL format with a tool of your choice like [CCMenu](http://ccmenu.org/):
```
Comment (RDIL) [inline docs/guide/writing-tasks.md]:
```suggestion
Use the following URL format with a tool of your choice (such as [CCMenu](http://ccmenu.org/)):
```

**Comment by fkorotkov** [comment]
Had some spare time after finishing a big refactoring. This is basically badges but in CCTray XML format. 😅

**Comment by bekh6ex** [comment]
This is very impressive!

> This is basically badges but in CCTray XML format

Does also set status to `Building` when the build is in progress?

**Comment by fkorotkov** [comment]
> I think this was the fastest implementation of a feature after a request I've ever seen!

Thank you for the awesome suggestion and feel free to created more feature requests!

**Pull request merged**